src/Http/Auth/Jwt/JsonWebToken.php

```diff
@@ -63,6 +63,11 @@ public function getType(): string
             return self::UIT_ID_V2_JWT_PROVIDER_TOKEN;
         }
 
+        // This is a check for Keycloak tokens, auth0 does not have this field
```

Comment:
```suggestion
        // Keycloak ID Tokens have a claim `typ` with a value of ID
```
I would give more info in the comment, because  Keycloak `accessTokens` also have a claim `typ` but with a different value.